.github/workflows/release.yml

```
@@ -0,0 +1,13 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    uses: guardian/gha-scala-library-release-workflow/.github/workflows/reusable-release.yml@main
+    permissions: { contents: write, pull-requests: write }
+    secrets:
+      SONATYPE_PASSWORD: ${{ secrets.AUTOMATED_MAVEN_RELEASE_SONATYPE_PASSWORD }}
```

Since https://github.com/guardian/gha-scala-library-release-workflow/pull/23 we need `TOKEN`, not `PASSWORD`...

```suggestion
      SONATYPE_TOKEN: ${{ secrets.AUTOMATED_MAVEN_RELEASE_SONATYPE_TOKEN }}
```

See https://github.com/guardian/etag-caching/blob/main/.github/workflows/release.yml#L11